src/WindowsTracing/KrabsTracer.cpp

```diff
@@ -270,6 +288,9 @@ void KrabsTracer::OutputStats() {
   ORBIT_LOG("Number of stack events: %u", stats_.num_stack_events);
   ORBIT_LOG("Number of stack events for target pid: %u", stats_.num_stack_events_for_target_pid);
   context_switch_manager_->OutputStats();
+  if (graphics_etw_provider_) {
```

Comment:
`graphics_etw_provider_ == nullptr` for the sake of consistency with the rest of the class.

Reply:
done